src/components/tasks/card/index.tsx

```diff
@@ -85,7 +86,16 @@ const Card: FC<Props> = ({
       });
     }
   }
-
+  
+  function updateTaskLevelItems(newTaskLevelValue: TaskLevelType){
+    const toChange: any = cardDetails;
```

Comment:
@Muralidhar22 Is line no 91 really required? If yes, then why is the type 'any'?

Reply:
yes it is required

Reply:
how? I feel we can directly use cardDetails and get the id instead of assigning cardDetails to another variable

Reply:
yes okay got your point